pkgs/tools/misc/cinc-client/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, ruby, bundlerApp, bundlerUpdateScript }:
+
+bundlerApp {
+  pname = "chef-bin";
```

Comment:
This should match the directory name, cinc-client

Reply:
The gem for cinc-client is called chef-bin upstream, so the evaluation fails with it set in that manner. I don't know if there is an easy way to fix this.